tests/lookup/tests.py

```diff
@@ -1029,6 +1029,13 @@ def test_regex_null(self):
         Season.objects.create(year=2012, gt=None)
         self.assertQuerySetEqual(Season.objects.filter(gt__regex=r"^$"), [])
 
+    def test_textfield_exact_null(self):
+        with self.assertNumQueries(1) as ctx:
+            self.assertSequenceEqual(Author.objects.filter(bio=None), [self.au2])
+        # Columns with IS NULL condition are not wrapped (except PostgreSQL).
+        bio_column = connection.ops.quote_name(Author._meta.get_field("bio").column)
+        self.assertIn(f"{bio_column} IS NULL", ctx.captured_queries[0]["sql"])
```

Comment:
From the ticket it looked like the Oracle setup was specific to the reporter's environment so just checking that we don't wrap IS NULL checks sounds fine to me 👍

Reply:
Also, `test_exact_none_transform()` would crash without this change as I reverted 1e4da439556cdd69eb9f91e07f99cf77997e70d2.